intelmq/bots/collectors/mail/collector_mail_attach.py

```diff
@@ -80,5 +108,13 @@ def process_message(self, uid, message):
         self.logger.info("Email report read.")
         return seen
 
+    @cache
+    def _gpg(self):
+        try:
+            from gnupg import GPG
+        except ImportError:
+            raise MissingDependencyError("python-gnupg", ">=0.5")
```

Comment:
I suggest doing the check in `init` (for quick user-feedback). If decyrption is activated, and the library is not installed, raise the exception.

Reply:
good point